app/bin/setup_kinto.py

```diff
@@ -31,51 +32,52 @@ def create_user(user, passw):
 print(f">>>> Creating kinto bucket: {os.environ['KINTO_BUCKET']}")
 print(
     client.create_bucket(
-        id=os.environ["KINTO_BUCKET"],
+        id=KINTO_BUCKET,
         permissions={"read": ["system.Everyone"]},
         if_not_exists=True,
     )
 )
 
 
-print(">>>> Creating kinto group: editors")
-print(
-    client.create_group(
-        id=f"{os.environ['KINTO_COLLECTION']}-editors",
-        bucket=os.environ["KINTO_BUCKET"],
-        data={"members": [f"account:{os.environ['KINTO_USER']}"]},
-        if_not_exists=True,
+for collection in os.environ["KINTO_COLLECTION"], os.environ["KINTO_COLLECTION_NIMBUS"]:
+    print(">>>> Creating kinto group: editors")
+    print(
+        client.create_group(
+            id=f"{collection}-editors",
+            bucket=KINTO_BUCKET,
+            data={"members": [f"account:{os.environ['KINTO_USER']}"]},
+            if_not_exists=True,
+        )
     )
-)
 
-print(">>>> Creating kinto group: reviewers")
-print(
-    client.create_group(
-        id=f"{os.environ['KINTO_COLLECTION']}-reviewers",
-        bucket=os.environ["KINTO_BUCKET"],
-        data={"members": [f"account:{REVIEW_USER}"]},
-        if_not_exists=True,
+    print(">>>> Creating kinto group: reviewers")
+    print(
+        client.create_group(
+            id=f"{collection}-reviewers",
+            bucket=KINTO_BUCKET,
+            data={"members": [f"account:{REVIEW_USER}"]},
+            if_not_exists=True,
+        )
     )
-)
 
-print(f">>>> Creating kinto collection: {os.environ['KINTO_COLLECTION']}")
-print(
-    client.create_collection(
-        id=os.environ["KINTO_COLLECTION"],
-        bucket=os.environ["KINTO_BUCKET"],
-        permissions={
-            "read": ["system.Everyone"],
-            "write": [
-                (
-                    f"/buckets/{os.environ['KINTO_BUCKET']}/groups/"
-                    f"{os.environ['KINTO_COLLECTION']}-editors"
-                ),
-                (
-                    f"/buckets/{os.environ['KINTO_BUCKET']}/groups/"
-                    f"{os.environ['KINTO_COLLECTION']}-reviewers"
-                ),
-            ],
-        },
-        if_not_exists=True,
+    print(f">>>> Creating kinto collection: {collection}")
+    print(
+        client.create_collection(
+            id=collection,
+            bucket=KINTO_BUCKET,
+            permissions={
+                "read": ["system.Everyone"],
+                "write": [
+                    (
+                        f"/buckets/{os.environ['KINTO_BUCKET']}/groups/"
```

Comment:
It's odd to go back/forth between using the `KINTO_BUCKET` var and the `os.environ` access to it.

Reply:
Oh good catch I missed that one 👍